packages/codegen-ui-react/lib/react-studio-template-renderer.ts

```diff
@@ -595,7 +597,7 @@ export abstract class ReactStudioTemplateRenderer extends StudioTemplateRenderer
       factory.createBindingElement(
         factory.createToken(ts.SyntaxKind.DotDotDotToken),
         undefined,
-        factory.createIdentifier(hasVariant ? 'restProp' : 'rest'),
+        factory.createIdentifier(hasBreakpoint ? 'restProp' : 'rest'),
```

Comment:
Are restProps only used for breakpoints?

Reply:
Yes, restProp is used to include a transition style onto components but it should only be added on components with a 'breakpoint' variant.